CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.14.21
```

Comment:
Should we do 5.15.0 considering the new package addition?

Reply:
Ideally, that package's version should have been different. No major addition has been done for other packages.

Reply:
Nevertheless, considering we release the packages as a "whole" this seems like a big feature enough to be highlighted in a minor release. Regardless of this, do we want to safe-gourd ourselves by releasing this as an alpha first? cc @siriwatknp

Reply:
@brijeshb42 I mentioned this because sometimes, if @mui/material has a feature requiring it to be a minor, we bump all other packages, so why not now?

Reply:
Ideally, nextjs package should have it's own version along with alpha release

Reply:
If we are syncing the versions I would expect this to release 5.15.0, if we are releasing it as alpha, we can merge as is.

Reply:
I would be happy with either of the above

Reply:
I think we can go with `5.15.0`.